src/SDKs/SqlManagement/Sql.Tests/ServerCommunicationLinkScenarioTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.ResourceManager;
+using Microsoft.Azure.Management.Sql;
+using Microsoft.Azure.Management.Sql.Models;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Sql.Tests
+{
+    public class ServerCommunicationLinkScenarioTests
+    {
+        [Fact]
+        public void TestCreateUpdateGetDropServer()
+        {
+            string testPrefix = "sqlcrudtest-";
+            string suiteName = this.GetType().FullName;
+            SqlManagementTestUtilities.RunTestInNewResourceGroup(suiteName, "TestCreateUpdateGetDropServer", testPrefix, (resClient, sqlClient, resourceGroup) =>
+            {
+                string server1Name = SqlManagementTestUtilities.GenerateName(testPrefix);
+                string login = "dummylogin";
+                string password = "Un53cuRE!";
+                string version12 = "12.0";
+                Dictionary<string, string> tags = new Dictionary<string, string>()
+                    {
+                        { "tagKey1", "TagValue1" }
+                    };
+
+                // Create two servers
+                var server1 = sqlClient.Servers.CreateOrUpdate(resourceGroup.Name, server1Name, new Server()
```

Comment:
this will require either re-recording or just manually fix the Names section at the bottom of recording file.